### ISSUE TITLE
Low-resolution textures

### ISSUE THREAD

**Issue by gigapouch** [nathan/phosphorus#315]
I love this project, but I have noticed a bug. The project below in the JavaScript player has lower resolution textures than the actual Scratch Project. Thanks for looking into this!

http://scratch.mit.edu/projects/26101065/
http://phosphorus.herokuapp.com/#26101065
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.111 Safari/537.36

**Comment by nathan** [comment]
Yeah, this is like https://github.com/nathan/phosphorus/issues/75. https://github.com/nathan/phosphorus/pull/280 fixes both, but still has issues.
